src/description.ts

```diff
@@ -1,21 +1,91 @@
-import { DEFAULT_LOCALE_DESCRIPTION } from './constants'
-import { Descriptions, LocalizedJSONSchema } from './types'
+import { JSONSchema6, JSONSchema6Definition } from 'json-schema'
+import { DEFAULT_DESCRIPTION_LOCALE } from './constants'
+import { Descriptions, LocalizedJSONSchema, LocalizedJSONSchemaDefinition } from './types'
 import { isNotEmpty } from './utils'
 
-export function getDescriptions (schema: LocalizedJSONSchema, localeDescription: string = DEFAULT_LOCALE_DESCRIPTION): Descriptions {
+export function getDescriptions (schema: LocalizedJSONSchema, descriptionLocale: string = DEFAULT_DESCRIPTION_LOCALE): Descriptions {
   const description = isNotEmpty(schema.description) ? schema.description : undefined
   const descriptions = schema.descriptions
 
   if (descriptions == null) {
-    return description != null ? { [localeDescription]: description } : {}
+    return description != null ? { [descriptionLocale]: description } : {}
   }
 
-  if (description != null && !Object.keys(descriptions).includes(localeDescription)) {
-    descriptions[localeDescription] = description
+  if (description != null && !Object.keys(descriptions).includes(descriptionLocale)) {
+    descriptions[descriptionLocale] = description
   }
   return descriptions
 }
 
-export function getLocalizedDescription (schema: LocalizedJSONSchema, locale: string, localeDescription?: string): string | undefined {
-  return getDescriptions(schema, localeDescription)[locale]
+export function getLocalizedDescription (schema: LocalizedJSONSchema, locale: string, descriptionLocale?: string): string | undefined {
+  return getDescriptions(schema, descriptionLocale)[locale]
+}
+
+function renderSimpleJsonSchema (locale: string, schema?: LocalizedJSONSchemaDefinition, descriptionLocale?: string): JSONSchema6Definition | undefined {
```

Comment:
Why `simple`?

Reply:
I named this function `renderSimpleJsonSchema` because there is already a `renderJsonSchema` and it calls it to render the schema definition if it's not a boolean. Should I rename it `renderJsonSchemaDefinition` (and maybe do the same with `renderArryOfJsonSchema` and `renderObjectOfJsonSchema`)?

Reply:
I don't think both function are required, they can be merged

Reply:
94ad7d56fc63f62e354895fa3ffebfc271070078